src/platform_linux/shmem.c

```diff
@@ -361,19 +393,17 @@ platform_shmcreate(size_t            size,
    }
 
    // Setup shared segment's control block at head of shared segment.
-   size_t      free_bytes;
    shmem_heap *shm = (shmem_heap *)shmaddr;
 
-   shm->shm_start          = shmaddr;
-   shm->shm_end            = (shmaddr + size);
-   shm->shm_next           = (shmaddr + sizeof(shmem_heap));
-   shm->shm_total_bytes    = size;
-   free_bytes              = (size - sizeof(shmem_heap));
-   shm->shm_free_bytes     = free_bytes;
-   shm->shm_used_bytes     = 0;
-   shm->shm_used_bytes_HWM = 0;
```

Comment:
The chunk of deleted inline code is now nicely replaced by a single call to `platform_save_usage_stats()` on L475, below.